core/db.py

```diff
@@ -52,7 +52,10 @@ def _execute_wrapper(self, sql, params, callback):
 
         start_time = time.time()
         try:
-            cur.execute(sql if self.type == self.SQLITE else sql.replace("?", "%s"), params)
+            if self.type == self.MYSQL:
+                sql = sql.replace("OR IGNORE", "IGNORE")
+                sql = sql.replace("?", "%s")
+            cur.execute(sql, params)
```

Comment:
since MySQL is documented better than SQLite, I have typically written SQL that is MySQL-flavored, and then done conversions to SQLite when necessary, rather than the other way around.  see https://github.com/Budabot/Tyrbot/blob/master/core/db.py#L119-L121 

also, the search and replace should be "INSERT OR IGNORE" rather than just "OR IGNORE" to prevent unintended changes

Reply:
Noted - now that you reminded where the other method was, should I move the "INSERT OR IGNORE" thing to the format_sql method? (pushing once you answered to this one)

Reply:
Yeah, actually, that might be a better place for it.  I don't see a reason not to do that.